src/modules/expression/literal/text.rs

```diff
@@ -39,8 +42,13 @@ impl TranslateModule for Text {
         let interps = self.interps.iter()
             .map(|item| item.translate(meta))
             .collect::<Vec<String>>();
-        let quote = meta.gen_quote();
-        format!("{quote}{}{quote}", translate_interpolated_region(self.strings.clone(), interps, true))
+        let strings = translate_interpolated_region(self.strings.clone(), interps, true);
+        if self.unquoted {
+            strings
```

Comment:
This is necessary so we generate Bash `for item in file*` not `for item in "file*"`.